apps/vaults/hooks/useSolverPortals.ts

```diff
@@ -150,7 +150,7 @@ export function useSolverPortals(): TSolverContext {
 		try {
 			const {tx: {value, gasLimit, ...rest}} = portals;
 			const transaction = await signer.sendTransaction({
-				value: BigNumber.from(value.hex),
+				value: BigNumber.from(value?.hex ?? 0),
```

Comment:
How come this is needed? From the [docs](https://docs.portals.fi/api/#tag/Portal/operation/PortalController_portal) it appears that those fields are required as a response

<img width="422" alt="Screenshot 2023-03-20 at 19 34 00" src="https://user-images.githubusercontent.com/78794805/226421449-0dc5da15-a2f9-4c12-b596-776ba02587b6.png">

Reply:
Yep, this is a mistake on the types due to some ethers weirdness. Will be fixed in the v2 version of the endpoints